frontend/packages/db-structure/src/parser/sql/postgresql/converter.ts

```diff
@@ -136,11 +198,64 @@ export const convertToSchema = (stmts: RawStmt[]): ProcessResult => {
     if (!tableName) return
 
     const columns: Columns = {}
+    const constraints: Constraints = {}
     for (const elt of createStmt.tableElts) {
       if ('ColumnDef' in elt) {
         const colDef = elt.ColumnDef
         if (colDef.colname === undefined) continue
 
+        let isPrimary = false
+        let isUnique = false
+        for (const constraint of colDef.constraints ?? []) {
```

Comment:
This for loop is to unify the logics with constraints check.
It refers to `colDef.constraints` objects to create constraint objects, relationship objects and make flag if the columns are primary or unique, which currently used to create `Column` object.